tools/respecDocWriter.js

```diff
@@ -249,9 +249,20 @@ function makeConsoleMsgHandler(page) {
    */
   return function handleConsoleMessages(whenToHalt) {
     page.on("console", async message => {
+      const text = message.text();
```

Comment:
This was replaced by stringifyJSHandle in 6a4b0eabaf413cd8d0a532d692b66ab35359ce7c because message.text() just returns `JSHandle@error` when the argument is `Error`.

Reply:
I don't know if they changed that too, but right now the old code returns nothing :( Give it a quick try:
```
node ./tools/respec2html.js -e --timeout 30 --src examples/basic.html --out /dev/null
```

Reply:
I also tried `map()`ing the JSHandles too for real errors, and got basically the same as `.text()`. Would you mind having a quick look?

Reply:
>but right now the old code returns nothing

Maybe because browser errors have no arguments, so `args.join(" ");` returns an empty string. We may ignore console messages which:

1. `message.args()` is empty
2. `message.text()` is not empty

What do you think?

Reply:
Yeah, I think that works

Reply:
I think we should brink it back, otherwise we'll have to reopen #1606.

Reply:
no probs, will bring it back.

Reply:
Ok, so, unwrapping via `stringifyJSHandle` still get's me to, for example, this array:

```
[ 'Document on track but no previous version: Add `previousMaturity`, and `previousPublishDate` to ReSpec\'s config.',
  'undefined' ]
```

However, `.text()` still gives us back the correct `text()`. I think Puppeteer must have themselves fixed what as causing #1606 (i.e., we don't need to unwrap).

Reply:
Oh, and it seems they changed "warn" to "warning" for type.